app/src/main/java/com/portfolio/tasky/entry/viewModels/LoginViewModel.kt

```diff
@@ -25,13 +25,13 @@ class LoginViewModel @Inject constructor(
 
 
 
-    fun emailChange(email : String){
+    fun emailChange(email : String) {
```

Comment:
I'd always try to name functions based on what they do. `emailChange` isn't really something you do. Might sound nitpicky here, but in larger codebases these things actually matter quite a lot, since new devs need to quickly understand parts of a codebase.
In this case, something like `onEmailChange()` or `submitEmailChange()` or so would make the purpose more clear 👍

Reply:
Okay

Done